app/repositories/ScratchRepository.scala

```diff
@@ -46,14 +46,13 @@ class ScratchRepositoryImpl @Inject() (mongoComponent: ReactiveMongoComponent, a
       idFormat = implicitly[Format[UUID]]
     )
     with ScratchRepository {
-  val timezone = DateTimeZone.forID(appConfig.scratchExpiryTZ)
 
   override def indexes: Seq[Index] = Seq(
     Index(Seq("expireAt" -> IndexType.Ascending), name = Some("expiryIndex"), options = BSONDocument("expireAfterSeconds" -> 0))
   )
 
   def save(process: JsObject): Future[RequestOutcome[UUID]] = {
-    val expiryTime = DateTime.now(timezone).withTime(appConfig.scratchExpiryHour, appConfig.scratchExpiryMinutes, 0, 0)
+    val expiryTime = ZonedDateTime.now.withHour(appConfig.scratchExpiryHour).withMinute(appConfig.scratchExpiryMinutes).withSecond(0)
```

Comment:
This lacks precision. We should change the withSecond to withSecond(59). If you're really keen we could also all withNano(999999999).

Reply:
Disagree, the requirements were expiry @ 23:59 not midnight - delta t ;)